app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
```

Comment:
the default policy for the index action is false - meaning it will never authorize successfully and the user will be redirected or not permitted access.  in the specific policy class for the resource (e.g. the WikiPolicy in this case), you need to overwrite this default policy by adding an index? method with some other logic that will determine whether or not to authorize (e.g. `user.present?`, etc).